spinalcordtoolbox/process_seg.py

```diff
@@ -193,7 +193,7 @@ def _properties2d(image, dim):
                   'eccentricity': region.eccentricity,
                   'orientation': orientation,
                   'solidity': solidity  # convexity measure
-    }
+                  }
 
```

Comment:
Nit, but perhaps this might be better off as:

```python
   # Fill up dictionary
    properties = {
        'area': area,
        'diameter_AP': diameter_AP,
        'diameter_RL': diameter_RL,
        'centroid': region.centroid,
        'eccentricity': region.eccentricity,
        'orientation': orientation,
        'solidity': solidity  # convexity measure
    }